test/index-test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const readFileHelper = require('../lib/filesystem.js');
+// const assert = require('assert');
+//
+// describe('Testing readFileHelper module', function(){
```

Comment:
@loganlsr best practice is to remove any comment blocks like this from your master branch - if you need this for reference, always feel free to create a testing branch (or series of dev/feature branches) and include comments there